src/doxy/Doxyfile

```diff
@@ -24,9 +24,9 @@ INPUT                  = $(DOXY_CONTENT)../../README.md $(DOXY_INPUT)
 LAYOUT_FILE            = $(DOXY_CONTENT)DoxygenLayout.xml
 USE_MDFILE_AS_MAINPAGE = README.md
 MAX_INITIALIZER_LINES  = 0
-PROJECT_NAME           = SAS JS
-PROJECT_LOGO           = $(DOXY_CONTENT)sasjs-logo.png
-PROJECT_BRIEF          = "Command line interface for SASjs"
+PROJECT_NAME           = $(PROJECT_NAME)
+PROJECT_LOGO           = $(DOXY_CONTENT)logo.png
+PROJECT_BRIEF          = $(PROJECT_BRIEF) PROJECT_REPO=$(PROJECT_REPO)
```

Comment:
I'm worried about this part - what if there is no github repository?  I really like the way that you have implemented it (nice workaround) but we could be setting ourselves up for problems if the user simply doesn't use / want to use GIT.

Maybe it would be safer / more robust (cause less confusion) to just have the $PROJECT_BRIEF here?